Traceback (most recent call last):
  File "/scratch/tyoeasley/brain_representations/src_py/run_lindecomp_dimstab.py", line 119, in <module>
    main(sys.argv)
  File "/scratch/tyoeasley/brain_representations/src_py/run_lindecomp_dimstab.py", line 75, in main
    run_dimensional_stability_tst(reps,output_basedir,permset,namelist,regval_list,decomp_method,n_workers)
  File "/scratch/tyoeasley/brain_representations/src_py/run_lindecomp_dimstab.py", line 109, in run_dimensional_stability_tst
    dim_vs_reg_list[i] = anl_pt.visualize_permtests(cancorrplots_outdir, data_outdir, nulldist_outdir, reglist_i)
  File "/mnt/beegfs/scratch/tyoeasley/brain_representations/src_py/analyze_CCA_permtests.py", line 18, in visualize_permtests
    null_idx = fname_nulllist.index(fname_datalist[i])
ValueError: 'NM_PROFUMO_and_Maps_PROFUMO.CCA_res' is not in list
